rnd/autogpt_server/autogpt_server/executor/scheduler.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, refresh_interval=10):
         self.refresh_interval = refresh_interval
 
     @property
-    def execution_manager_client(self):
+    def execution_manager_client(self) -> ExecutionManager:
```

Comment:
This is unrelated but somehow broken due to the introduction of user_id on execution cc @aarushik93

Reply:
what is broken on it? can you explain?

Reply:
The execution_manager.add_execution becomes 3 paramater function and this code still call using two